ingeniamotion/disturbance.py

```diff
@@ -119,11 +119,14 @@ def map_registers(self, registers):
             int: max number of samples
 
         Raises:
+            IMDisturbanceError: If the register list is empty.
             IMDisturbanceError: If the register is not allowed to be mapped as
                 a disturbance register.
         """
         if not isinstance(registers, list):
             registers = [registers]
+        if len(registers) == 0:
+            raise IMDisturbanceError("registers list is empty")
```

Comment:
I'm not sure if the exception should be an IMDisturbanceError or a ValueError, because the parameter is empty and this value is not allowed. What do you think?